### ISSUE TITLE
wont play m3u radio file

### ISSUE THREAD

**Issue by magiva** [giachello/netradio#5]
i added Swedish radio station p3 as follows 
https://www.radio-browser.info/history/1d35c168-deb1-4751-a1e6-ae19fb3e7a3e

Url: https://sverigesradio.se/topsy/direkt/164-hi-mp3.m3u

but nothing i try plays it, i can play it with vlc, i can play it on the radio-browser but not in home assistant

**Comment by giachello** [comment]
yeah, the HA media player doesn't do M3U, we should add support to pull out the actual stream link from the M3U file and pass it on to the media player. Don't have time right now to do it, but if you are so inclined please go ahead!

**Comment by magiva** [comment]
ok, i pulled the detail out with vlc and added it that way, leaving this
comment so others might want to do the same...maybe you could add that to
your front page notes for others
VLC: tools \ media info (control i)
